src/tests/modules/bson.rs

```diff
@@ -265,6 +267,34 @@ fn from_external_datetime() {
         assert_millisecond_precision(from_chrono);
     }
 
+    {
```

Comment:
can we put these in their own separate test (something like `from_datetime_builder`)? it's a little easier to debug failures that way

Reply:
Fixed!